.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3snapshot
```

Comment:
should it be `nightly`?

Reply:
nightly is 7.4 :see_no_evil:

Reply:
lets see https://twitter.com/markusstaab/status/1044500936415219712

Reply:
My work is based on https://github.com/travis-ci/travis-ci/issues/9717

Reply:
thx for the hint. does it `require: sudo` then?

https://github.com/travis-ci/travis-ci/issues/9717#issuecomment-421056032

Reply:
this is the current stage: https://github.com/sabre-io/dav/pull/1080/files